homeassistant/components/gpsd/sensor.py

```diff
@@ -116,7 +111,7 @@ def extra_state_attributes(self) -> dict[str, Any]:
             ATTR_LATITUDE: self.agps_thread.data_stream.lat,
             ATTR_LONGITUDE: self.agps_thread.data_stream.lon,
             ATTR_ELEVATION: self.agps_thread.data_stream.alt,
-            ATTR_GPS_TIME: self.agps_thread.data_stream.time,
+            ATTR_TIME: self.agps_thread.data_stream.time,
```

Comment:
This will make the PR breaking. Is there a reason to change this?

Reply:
Just for cleanup. If you worry about that I'll revert it back.

Reply:
If I revert both there will not much left ;)
I'll close this PR and add another one for the coordinator.